commands/test/test.go

```diff
@@ -56,13 +56,22 @@ func NewTestCommand(osExit func(int), getOutputManager func() OutputManager) *co
 			}
 		},
 		Run: func(cmd *cobra.Command, fileList []string) {
-			if len(fileList) < 1 {
+
+			// Remove any blank files from the array
+			var nonBlankFileList []string
```

Comment:
On the surface, the code is all fine, but want to get some clarifications to make sure we're doing the right thing.

The direct question would be: Do we want to consider this a CLI feature or a feature of the library? Right now, it's only implemented for the `test` command. If a package consumer passed a `fileList` containing `""` to `GetConfigurations`, they'd run into the same problem. Which leads me to believe we should move this down into core library functionality, so that all package consumers and `conftest` commands have this.

Making this a library feature means we should just be able to add a blank check and a `continue` inside of the `for..range`

If we want it to be purely a CLI feature, we should probably also add this to the `parse` command for consistency.